Samples/Shared/CameraKsPropertyHelper/KSHelper.h

```diff
@@ -6,6 +6,8 @@
 
 namespace winrt::CameraKsPropertyHelper::implementation
 {
+    static GUID KSPROPERTYSETID_WindowsStudioCameraControl = 

```

Comment:
>KSPROPERTYSETID_WindowsStudioCameraControl [](http://example.com/codeflow?start=16&length=42)

This should be available in ks.h if newer SDK version is used. Named KSPROPERTYSETID_WindowsCameraEffect, so maybe this should use same name?